font.go

```diff
@@ -85,6 +91,8 @@ func NewFont(family string, pointSize int, style FontStyle) (*Font, error) {
 	return font, nil
 }
 
+const fontWeightSemiLight = 350
```

Comment:
Maybe this?
```suggestion
const fontWeightSemiLight = (win.FW_LIGHT + win.FW_NORMAL) / 2
```
It's not super clear where the magic number of 350 comes from.